Source/Painting/SvgDeferredPaintServer.cs

```diff
@@ -1,45 +1,40 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using System.Linq;
 using System.Drawing;
 
 namespace Svg
 {
     /// <summary>
-    /// A wrapper for a paint server which isn't defined currently in the parse process, but
-    /// should be defined by the time the image needs to render.
+    /// A wrapper for a paint server which isn't defined currently in the parse process,
+    /// but should be defined by the time the image needs to render.
     /// </summary>
     public class SvgDeferredPaintServer : SvgPaintServer
     {
         private bool _serverLoaded = false;
         private SvgPaintServer _concreteServer;
 
-        public SvgDocument Document { get; set; }
         public string DeferredId { get; set; }
 
         public SvgDeferredPaintServer() { }
-        public SvgDeferredPaintServer(SvgDocument document, string id)
+        public SvgDeferredPaintServer(string id)
```

Comment:
This changes the public API - I'm a bit reluctant to make this kind of changes.
I would refer this to @tebjan to comment/approve.

Reply:
@mrbean-bremen 

Thank you for your review.
I agree.
I will add new constructor.
I will add ‘Obsolete‘  this and not use ‘document‘ parameter.